### PR TITLE
Fixes new layout of public_key file

### PR DESCRIPTION
**Pull request by vpetersson**
Hotfix for new layout of `public_keys` file

### CODE REVIEW

File: hw_diag/tests/test_get_public_keys.py [hunk 1]
```diff
@@ -6,24 +6,21 @@
...
 class TestGetPublicKeys(unittest.TestCase):
-    TEST_DATA = [
-        "{pubkey,",
-        '"112mPWkGW55kcbQTgbtJvgAKMSTrEhHgavrdF1Cbu8FU85tTL4Nc"}.',
-        '{onboarding_key,',
-        '"112mPWkGW55kcbQTgbtJvgAKMSTrEhHgavrdF1Cbu8FU85tTL4Nc"}.',
-        '{animal_name,',
-        '"gigantic-basil-turtle"}.'
-    ]
+    TEST_DATA = '{}\n{}\n{}'.format(
```
Comment (marvinmarnold) [inline hw_diag/tests/test_get_public_keys.py:9]: I'm unsure about the line breaks. The miner I'm looking at doesn't appear to have any.

**Review by marvinmarnold — APPROVED** [review]
The code looks internally consistent. But I don't quite understand the context. Do you have a cross-link to the code that generates the `public_keys` file?

File: hw_diag/tests/test_get_public_keys.py (hunk 1 above)
Comment (vpetersson) [reply hw_diag/tests/test_get_public_keys.py:9]: It was introduced in the latest GA. We've had half the fleet fall over because of this.

**Review by robputt — APPROVED** [review]

File: hw_diag/tests/test_get_public_keys.py (hunk 1 above)
Comment (vpetersson) [reply hw_diag/tests/test_get_public_keys.py:9]:
```
/tmp/build # wc -l /var/data/public_keys
3 /var/data/public_keys
```

**Comment by vpetersson** [comment]
> The code looks internally consistent. But I don't quite understand the context. Do you have a cross-link to the code that generates the `public_keys` file?

Yeah, so this is just really a band aid while we wait for @robputt to adopt the internal API. I really hate how we shell out and read/parse random files on disk. It's just asking for trouble.

**Comment by shawaj** [comment]
> The code looks internally consistent. But I don't quite understand the context. Do you have a cross-link to the code that generates the `public_keys` file?

https://github.com/NebraLtd/hm-miner/blob/master/start-miner.sh and https://github.com/helium/miner/blob/master/src/miner_keys.erl

**Pull request merged**